__snapshots__/src/tree/index.test.js.md

```diff
@@ -1,13 +0,0 @@
-# Snapshot report for `src\tree\index.test.js`
```

Comment:
Do you know why this file is showing up as deleted? Based on the changes, there should have been no changes to this snapshot.

Reply:
You've  or someone has set skip on the snapshot-tests for related file meaning no snapshots are generated. Was not aware of the --update-snapshots command at first so just deleted and checked changes. I let them be deleted since I saw that they're were skipped.